content/en/docs/Configuration/namespace-management.md

```diff
@@ -5,188 +5,201 @@ description: "Configuring the namespaces accessible and visible to Kiali."
+{{% /alert %}}
 
 {{% alert color="warning" %}}
-As of Kiali 1.67, the following settings are deprecated:
+As of Kiali 2.0, the following settings are no longer supported:
+* deployment.accessible_namespaces
 * api.namespaces.exclude
 * api.namespaces.include
 * api.namespaces.label_selector_exclude
 * api.namespaces.label_selector_include
+{{% /alert %}}
+
+## Cluster Wide Access Mode
+
+By default, the Kiali Server is given cluster-wide access to all namespaces on the local cluster. This is controlled by the Kiali CR setting `deployment.cluster_wide_access`, which has a default value of `true` when not specified.
+
+For clusters that have multiple Kiali Servers installed, only one of them may be installed with `deployment.cluster_wide_access` set to `true`.
 
-It is recommended to instead use Istio Discovery Selectors to limit the namespaces in the mesh.
+In order to restrict the Kiali Server so that it only has access to certain namespaces on the local cluster, it must first have its cluster-wide access disabled. You do this by setting `deployment.cluster_wide_access` to `false` in the Kiali CR.
+
+{{% alert color="info" %}}
+You can still use discovery selectors (explained below) to limit what Kiali will make visible in the UI while `cluster_wide_access` remains `true`. You would want to do this for the performance benefits it provides the Kiali Server. But with this, the Kiali Server will be granted ClusterRole permissions rather than individual Role permissions per namespace. In other words, it will have _access_ to all namespaces, but will not make all of them _visible_.
 {{% /alert %}}
 
 ## Accessible Namespaces
 
-The "accessible namespaces" setting configures which namespaces are accessible to the Kiali server itself.
-The accessible namespaces are defined by a list of regex expressions that match against the namespace names
-visible to the Kiali operator. If left unset, the Kiali server will be given access to all namespaces in
-the cluster.
+With cluster-wide access disabled, the Kiali Server must be told what namespaces are accessible to it.  These accessible namespaces are defined by a list of discovery selectors that match namespaces.
 
-The list of accessible namespaces is specified in the Kiali CR via the
-`accessible_namespaces` setting, under the main `deployment` section. As an
-example, if Kiali is to be installed in the `istio-system` namespace, and is
-expected to monitor all namespaces prefixed with `mycorp_`, the setting would
-be:
+The list of accessible namespaces is specified in the Kiali CR via the `deployment.discovery_selectors.default` setting. As an example, if Kiali is to be installed in the `istio-system` namespace, and is expected to monitor all namespaces with the label "my-mesh", the setting would be:
```

Comment:
Does Kiali's install namespace need to be labeled?

Reply:
no, because later the docs say (line 57):

`The Kiali Operator will always give the Kiali Server access to the namespace where the Kiali Server is installed and to the Istio control plane namespace (which may be the same namespace), whether those namespaces match a discovery selector or not.`